estate/models/__init__.py

```diff
@@ -0,0 +1,7 @@
+from . import estate_property
+from . import estate_property_type, estate_property_tag, estate_property_offer
+
+assert estate_property
+assert estate_property_type
+assert estate_property_tag
+assert estate_property_offer
```

Comment:
```suggestion
from . import estate_property
from . import estate_property_type
...
```
nitpick: this syntax is fine but in odoo code, `__init__` files usually look like this